commit 2ca83509dbc72dfdfa9cc7103c2b29db31e07f3a
Author: Heiko Becker <heirecka@exherbo.org>
Date:   Sun Mar 29 12:53:00 2020 +0200

    Fix build with Qt 5.15
    
    QPainterPath is no longer included via qtransform.h (since
    5.15.0-beta2, 50d2acdc93b4de2ba56eb67787e2bdcb21dd4bea in qtbase.git).

diff --git a/src/lib/tools/qztools.cpp b/src/lib/tools/qztools.cpp
index 499b2250..04f19b71 100644
--- a/src/lib/tools/qztools.cpp
+++ b/src/lib/tools/qztools.cpp
@@ -25,6 +25,7 @@
 #include <QByteArray>
 #include <QPixmap>
 #include <QPainter>
+#include <QPainterPath>
 #include <QBuffer>
 #include <QFile>
 #include <QDir>
